components/Table/Table.tsx

```diff
@@ -0,0 +1,256 @@
+import * as React from 'react'
+import { useState, useRef, useMemo, useEffect } from 'react'
+// import DataService from "./services/TutorialService";
+import { useTable } from 'react-table'
+import Pagination from '@material-ui/lab/Pagination'
+
+interface ITableProps {
+  history: any
+  DataService: any
+  pageSizes: any
+  columns: any
+  costumeClassName: any
+}
+const Table = (props: ITableProps) => {
+  const [tutorials, setTutorials] = useState([])
+  const [searchTitle, setSearchTitle] = useState('')
+  const tutorialsRef: any = useRef()
```

Comment:
use correct type 
I think you can use this type
React.RefObject<HTMLElement>

Reply:
I can't find better way for edit type of props but refactor this code again and again thanks